mscore/data/shortcuts-Mac.xml

```diff
@@ -21,7 +29,7 @@
     <std>63</std>
     </SC>
   <SC>
-    <key>file-close</key>
+    <key>file-close-tab</key>
```

Comment:
why that renaming? That would be an incompatible change, wouldn't it?

Reply:
@Jojo-Schmitz The new name is more descriptive of the shortcut's actual function, especially now that a new "Close Window" option is available in the same File menu. I'm not sure exactly what you mean by an incompatible change, unless you mean the change should be reflected in the non-mac shortcuts XML file, which I now realize should be done next. How can I modify the PR, or should I somehow open a new PR?

Reply:
The name is not visible to any user, just to programmers, but the shortcut file would become incompatible.
You can change this in a new comment, then squash the commits. No need for a new PR.

Reply:
@Jojo-Schmitz I see what you're saying, however after reviewing some of the codebase, it's evident that there is opportunity for improving the ease of code readability through more descriptive naming. Since I changed the user-visible option to Close Tab, I thought it best to modify it for developers working on the backend. 

In regards to compatibility, I already updated the code that references the shortcut name (in Musescore.cpp), and it seems to work as expected. If it's a deal breaker for merging the PR, then I can change it back, but I believe it to be an enhancement, not a detraction.

Reply:
Just use `git push -f`` as indicated